src/lib/progress-bar/progress-bar.ts

```diff
@@ -82,6 +94,18 @@ export class MatProgressBar extends _MatProgressBarMixinBase implements CanColor
   set bufferValue(v: number) { this._bufferValue = clamp(v || 0); }
   private _bufferValue: number = 0;
 
+  @ViewChild('primaryValueBar') primaryValueBar: ElementRef;
```

Comment:
This property should be prefixed with an underscore to indicate that it is not part of the public API surface

Reply:
done